### PR TITLE
feat: Add signMessage debug interface

### PR DESCRIPTION
**Pull request by sprohaszka-ledger**
Add the capacity of sending a message (a string) to sign.

Depends on:
- https://github.com/LedgerHQ/ledger-live/pull/602
- https://github.com/LedgerHQ/live-app-sdk/pull/118
- https://github.com/LedgerHQ/ledger-live/pull/701

### CODE REVIEW

File: src/components/DebuggerV2/handlers.tsx [hunk 1]
```diff
@@ -156,5 +156,24 @@ export function getHandlers(platformSDK: LedgerLiveApi): MethodHandler[] {
...
+      inputs: [
+        {
+          name: "account",
+          component: AccountSelector,
+        },
+        {
+          name: "message",
+          component: JSONTextArea,
+          modifier: (value: any) => value,
+          defaultValue: "Message de test",
```
Comment (chabroA) [inline src/components/DebuggerV2/handlers.tsx]:
```suggestion
          defaultValue: "Test message",
```

File: src/components/DebuggerV2/handlers.tsx [hunk 2]
```diff
@@ -156,5 +156,24 @@ export function getHandlers(platformSDK: LedgerLiveApi): MethodHandler[] {
...
+      handler: platformSDK.signMessage.bind(platformSDK),
+      inputs: [
+        {
+          name: "account",
+          component: AccountSelector,
+        },
+        {
+          name: "message",
+          component: JSONTextArea,
+          modifier: (value: any) => value,
```
Comment (chabroA) [inline src/components/DebuggerV2/handlers.tsx]: The `JSONTextArea` component could be turned in a basic `TestArea` component and the `mode` (like `json`) could be passed as a prop

**Review by IAmMorrow — APPROVED** [review]

File: src/components/DebuggerV2/handlers.tsx (hunk 2 above)
Comment (IAmMorrow) [reply src/components/DebuggerV2/handlers.tsx]: Indeed

**Review by Justkant — APPROVED** [review]

**Review by chabroA — APPROVED** [review]

File: src/components/DebuggerV2/handlers.tsx [hunk 3]
```diff
@@ -171,7 +171,7 @@ export function getHandlers(platformSDK: LedgerLiveApi): MethodHandler[] {
         {
           name: "message",
           component: JSONTextArea,
-          modifier: (value: any) => value,
+          modifier: (value: any) => Buffer.from(value),
```
Comment (Justkant) [inline src/components/DebuggerV2/handlers.tsx:174]: So this version of the debugger will only work with new version of the platform sdk we need to sync versions between this repo and the LLD/LLM using the `apiVersion` field in the manifest of the debugger

**Review by Justkant — APPROVED** [review]

**Pull request merged**